src/main/java/seedu/address/logic/parser/IssueCliSyntax.java

```diff
@@ -10,6 +10,7 @@ public class IssueCliSyntax {
     public static final Prefix PREFIX_DEADLINE = new Prefix("d/");
     public static final Prefix PREFIX_PRIORITY = new Prefix("p/");
     public static final Prefix PREFIX_PROJECT_ID = new Prefix("pid/");
+    public static final Prefix PREFIX_PROJECT_NAME = new Prefix("pn/");
```

Comment:
is this necessary? Would PREFIX_NAME of `/n` work as well. Would be nice to cut down prefix uses if possible